### PR TITLE
Format codes based on .editorconfig, Part 8

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/convert1.vb [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
+                                              byteValue2)
+        Catch e As OverflowException
+            Console.WriteLine("Unable to convert the {0} value {1}.",
+                                              sbyteValue.GetType().Name, sbyteValue)
+        End Try
+        ' The example displays the following output:
+        '       Converted the Int32 value 12534 to the Decimal value 12,534.00.
+        '       Converted the Byte value 255 to the Int32 value 255.
+        '       Converted the Double value 1.632513E+013 to the Int64 value 16,325,130,000,000.
+        '       Unable to convert the SByte value -16.
```
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/convert1.vb:64]:
```suggestion
        '       Unable to convert the SByte value -16.
```

Could we remove this too?

File: samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/convert1.vb [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+            Console.WriteLine("{0} converted to {1}.",
+                                              doubleValue, integerValue)
+        Catch e As OverflowException
+            Console.WriteLine("Unable to convert {0} to an integer.",
+                                              doubleValue)
+        End Try
+        ' The example displays the following output:
+        '       13956810.96702888123451471211 converted to 13956810.9670289.
+        '       42.72 converted to 43.
+        ' </Snippet9>
```
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/convert1.vb:89]:
```suggestion
        ' </Snippet9>
```

File: samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/implicit1.vb [hunk 3]
```diff
@@ -2,85 +2,85 @@
...
+        ' <Snippet3>
+        Dim sbyteValue As SByte = -120
+        Dim value As ByteWithSign = sbyteValue
+        Console.WriteLine(value.ToString())
+        value = Byte.MaxValue
+        Console.WriteLine(value.ToString())
+        ' The example displays the following output:
+        '       -120
+        '       255
+        ' </Snippet3>
```
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.conversion/vb/implicit1.vb:59]:
```suggestion
        ' </Snippet3>
```

**Review by IEvangelist — CHANGES_REQUESTED** [review]
This looks good, thank you. We'll :shipit: when you make the suggested changes.

**Comment by NextTurn** [comment]
So it seems that the formatter doesn't trim white space in comments.

**Comment by IEvangelist** [comment]
> So it seems that the formatter (or Roslyn, to be exact) doesn't trim white space in comments.

I think you're probably correct. How exactly are you applying the *.editorconfig* rules to update the code?

**Review by IEvangelist — APPROVED** [review]

**Comment by NextTurn** [comment]
> How exactly are you applying the _.editorconfig_ rules to update the code?

I'm using *dotnet-format* since the very first PR of this series: dotnet/samples#2865.

**Pull request merged**